src/meta/client/MetaClient.cpp

```diff
@@ -1746,9 +1746,6 @@ MetaClient::regConfig(const std::vector<cpp2::ConfigItem>& items) {
 
 folly::Future<StatusOr<std::vector<cpp2::ConfigItem>>>
 MetaClient::getConfig(const cpp2::ConfigModule& module, const std::string& name) {
-    if (!configReady_) {
```

Comment:
How about judge `skipConfig_` instead of just delete it? This is useful

Reply:
I think it's safe and correct if delete it, the meta should process request and return correctly in any cases.